### PR TITLE
Add nmap and iputils installations to Docker file

### PR DESCRIPTION
**Pull request by YoyoTT**
*Issue #, if available:*

*Description of changes:*
Add nmap and iputils installation to Docker file

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/aws/eks-anywhere/docker/linux/Dockerfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ ARG TARGETARCH
 ARG TARGETOS
 
 RUN yum install tar -y && \
+    yum install nmap -y && \
```
Comment (abhay-krishna) [inline projects/aws/eks-anywhere/docker/linux/Dockerfile:8]: You can input multiple package names to the same install command, as follows:
```suggestion
    yum install -y iputils nmap tar && \
```

File: projects/aws/eks-anywhere/docker/linux/Dockerfile [hunk 2]
```diff
@@ -5,6 +5,7 @@ ARG TARGETARCH
 ARG TARGETOS
 
 RUN yum install tar -y && \
+    yum install -y nmap iputils tar && \
```
Comment (abhinavmpandey08) [inline projects/aws/eks-anywhere/docker/linux/Dockerfile:8]: tar is specified twice
```suggestion
RUN yum install -y nmap iputils tar && \
```

**Review by abhinavmpandey08 — APPROVED** [review]

**Pull request merged**